plugins/liquid/refined-types-plugin/src/main/kotlin/arrow/meta/plugins/liquid/phases/analysis/solver/check/Expressions.kt

```diff
@@ -696,6 +697,13 @@ private fun SolverState.checkConstantExpression(
   NoReturn
 }
 
+private fun String.toBigIntegerSeveralAttempts(): BigInteger? =
```

Comment:
LOL what's this xD?

Reply:
Me after figuring out that after 30 minutes debugging it was only a matter of parsing an integer correctly xD

I've changed it to `asIntegerLiteral` and made it follow Kotlin's grammar.

Reply:
But maybe it should be replaced with https://github.com/JetBrains/kotlin/blob/master/compiler/psi/src/org/jetbrains/kotlin/parsing/ParseUtils.kt altogether